assets/reader-activation-auth/style.scss

```diff
@@ -7,6 +7,10 @@
 	input[type='email'].nphp {
 		@include mixins.visuallyHidden;
 	}
+
+	.response-container {
+		margin-bottom: 12px;
```

Comment:
I was going to comment that this could use the Newspack UI variables, then realized it's using a completely different CSS file 😅 

This is probably a question that goes beyond this PR, but is this stylesheet needed in the new modals? When I remove the `newspack-reader-auth` class from the modal and it doesn't seem to change the styles but I may be missing something! 

Just to keep stuff together, I think it may make sense to make a non-error version of the `newspack-ui__inline-error` that got removed from this element, and use that to handle the spacing in the Newspack UI styles instead -- what do you think?